client/src/redux/types.ts

```diff
@@ -1,5 +1,14 @@
-import { FlashApp, FlashMessageArg } from '../components/Flash/redux';
-import { MainApp } from '.';
+import { FlashMessages } from '../components/Flash/redux/flash-messages';
+import { ns as MainApp } from './action-types';
+
+// Shaun: Is this fine as in "rootReducer" you imported both reducer and namespace together from same file for easier updation
```

Comment:
@ShaunSHamilton ?

Reply:
Good question... give me a minute on this.

Reply:
This seems fine. Nothing wrong with it.

Ideally, I imagine we would export all state apps from a single path. This file seems like it could suit.

Reply:
I.e. we define both common types and namespace constants here? Sounds reasonable.

On reflection, we could do that and my suggestions separately. @Nirajn2311 is that okay?